src/backend/BinaryOperator.js

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
We might want to create a folder/module for nodes under backend instead of throwing everything directly under.

Reply:
Agree!